devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy

```diff
@@ -335,3 +349,23 @@ private shouldUseOldBatteryReporting() {
 
 	return isFwVersionLess // If f/w version is less than 1.15.7 then do NOT smooth battery reports and use the old reporting
 }
+
+private Boolean isFrientSensor() {
+	device.getDataValue("manufacturer") == "frient A/S"
+}
+
+private Map temperatureEndpoint() {
+	if (isFrientSensor()) {
+		[destEndpoint: 0x26]
+	} else {
+		[:]
+	}
+}
+
+private Map powerEndpoint() {
+	if (isFrientSensor()) {
+		[destEndpoint: 0x23]
+	} else {
+		[:]
+	}
+}
```

Comment:
too late to point this out now, but these calls only ever occur after we've already checked for `isFrientSensor` so they should have just been inlined I think

Reply:
Steven I thought about that initially but then realized that separating the endpoint mappings would lead to a cleaner way to reuse them across API's in future as more devices are added. So one could just have single call to `zigbee.configureReporting` for multiple devices with the same configuration requirement but with different endpoints without any additional conditions like `isFrientSensor`

Reply:
The usual way we handle this is essentially to cross that bridge when we get to it. The double-call to the model check really rubbed me the wrong way so I've already changed it in both DTHs.